src/Pages/CardReader/CardReader.js

```diff
@@ -32,6 +32,7 @@ export default function CardReader() {
   const [paginationText, setPaginationText] = useState('');
   const [loading, setLoading] = useState(false);
   const [total, setTotal] = useState(0);
+  const [selected, setSelected] = useState('Card Registery');
```

Comment:
```suggestion
  const [selected, setSelected] = useState('Card Registry');
```
spelling mistake lol - we wanna make sure this is consistent so that there are no silly errors later